.ci/jenkins/Jenkinsfile.bump-up-version

```diff
@@ -0,0 +1,100 @@
+import org.jenkinsci.plugins.workflow.libs.Library
+@Library('jenkins-pipeline-shared-libraries')_
+
+import org.kie.jenkins.MavenCommand
+
+def repoName = 'kie-benchmarks'
+
+pipeline {
+    agent {
+        label 'kie-rhel8 && !built-in'
+    }
+    tools {
+        maven env.BUILD_MAVEN_TOOL
+        jdk env.BUILD_JDK_TOOL
+    }
+
+    environment {
+        BRANCH_HASH = "${util.generateHash(10)}"
+    }
+
+    options {
+        timestamps()
+        timeout(time: 30, unit: 'MINUTES')
+    }
+
+    stages {
+        stage('Initialize') {
+            steps {
+                script {
+                    cleanWs()
+
+                    dir("${repoName}") {
+                        checkoutRepo("${repoName}", "${getTargetBranch()}")
+                    }
+                }
+            }
+        }
+        stage('bump up version') {
+            steps {
+                script {
+                    dir("${repoName}") {
+                        String commitMsg = "bumped up ${getDependencyProject()} version to next SNAPSHOT"
+                        String localBranch = "update-${getDependencyProject()}-${env.BRANCH_HASH}"
+                        githubscm.createBranch(localBranch)
+                        maven.mvnSetVersionProperty("version.org.${getDependencyProject()}", getVersion())
+                        String prLink = commitAndCreatePR(commitMsg, localBranch, "${getTargetBranch()}")
+                        sh "git checkout ${getTargetBranch()}"
+                        mergeAndPush(prLink, "${getTargetBranch()}")
+                        githubscm.removeRemoteBranch('origin', localBranch, getGitAuthorCredsId())
+                    }
+                }
+            }
+        }
+    }
+    post{
+        always{
+            cleanWs()
+        }
+    }
+}
+
+String getVersion() {
+    return params."${getDependencyProject().toUpperCase()}_VERSION"
```

Comment:
I just think that you don't even need the dependency name for the parameter.
The parameter could be just named something like `NEW_VERSION`